packages/server/database/migrations/1644229555565_following_user_followed_user.ts

```diff
@@ -0,0 +1,30 @@
+import BaseSchema from '@ioc:Adonis/Lucid/Schema';
+
+export default class FollowingUserFollowedUser extends BaseSchema {
+    protected tableName = 'following_user_followed_user';
+
+    //pivot table
+    public up(): void {
+        this.schema.createTable(this.tableName, (table) => {
+            table.increments('id');
```

Comment:
Do you think we should use a UUID auto-generated instead of an integer?

Reply:
I understand and share your concerns, it might be more logical and idiomatic to use uuid here too.
I've been using an integer as primary and ID here because we've also been doing so for the `mpeRoom` pivot table.
Can't remember if this was a defined choice or just a forgotten detail ? 
This might also be a way to differentiate pivot from classic tables

Reply:
Currently when we had a table with an UUID as the primary key, we were generating the UUID in JS code. The issue here is that I don't think we would be able to create the primary key like that.
As a consequence, we would need to use a function such as `uuid_generate_v4` (https://www.postgresql.org/docs/current/uuid-ossp.html) in the `DEFAULT` case of the primary key column. This would introduce a dependency to PostgreSQL as I do not think this function is available in other databases.

Using a `SERIAL` here, as you do, seems a lot easier 😄